tests/constructor-bottom/constructor-bottom.helper.js

```diff
@@ -0,0 +1,305 @@
+            message
+          }
+        }
+      }
+    `,
+    variables: {
+      id: wrongID,
+      constructorElement: addConstructor,
+    },
+  });
+  return updateConstructor.data.updateConstructorBottom.message;
+};
+const deleteConstructorBottom = async (constructorId, operations) => {
+  const deletedConstructor = await operations.mutate({
+    mutation: gql`
+      mutation($id: ID!) {
+        deleteConstructorBottom(id: $id) {
+          ... on ConstructorBottom {
+            _id
+          }
+        }
+      }
+    `,
+    variables: { id: constructorId },
+  });
+  return deletedConstructor.data.deleteConstructorBottom._id;
+};
+const getConstructorBottom = async (newConstructorForQuery, operations) => {
+  const constructorBottomById = await operations.query({
+    query: gql`
+      query($id: ID!) {
+        getConstructorBottomById(id: $id) {
+          ... on ConstructorBottom {
+            _id
+            name {
+              lang
+              value
+            }
+            material {
+              _id
+            }
+            image
+            color {
+              _id
+            }
+            available
+            default
+          }
+          ... on Error {
+            statusCode
+            message
+          }
+        }
+      }
+    `,
+    variables: { id: newConstructorForQuery },
+  });
+  return constructorBottomById.data.getConstructorBottomById;
+};
+const constructorBottomByIdEr = async (wrongID, operations) => {
+  const constructorBottomById = await operations.query({
+    query: gql`
+      query($id: ID!) {
+        getConstructorBottomById(id: $id) {
+          ... on ConstructorBottom {
+            _id
+            name {
+              lang
+              value
+            }
+            material {
+              _id
+            }
+            image
+            color {
+              _id
+            }
+            available
+            default
+          }
+          ... on Error {
+            statusCode
+            message
+          }
+        }
+      }
+    `,
+    variables: { id: wrongID },
+  });
+  return constructorBottomById.data.getConstructorBottomById;
+};
+const getAllConstructorBottom = async operations => {
+  const allConstructorBottom = await operations.query({
+    query: gql`
+      query($limit: Int, $skip: Int) {
+        getAllConstructorBottom(limit: $limit, skip: $skip) {
+          items {
+            _id
+            name {
+              lang
+              value
+            }
+            material {
+              _id
+              name {
+                lang
+                value
+              }
+              purpose
+              available
+            }
+            color {
+              _id
+              colorHex
+            }
+            image
+          }
+        }
+      }
+    `,
+  });
+  return allConstructorBottom.data.getAllConstructorBottom.items;
+};
+
+const updateConstructorBottomEr = async (
```

Comment:
You already have `updateConstructorBottom `for this

Reply:
done